requirements.txt

```diff
@@ -0,0 +1,3 @@
+django==1.8
+ipdb
+nose
```

Comment:
nose and ipdb would be better to be included in another requirements file eg. dev.txt as they are not requirements for the live system but for testing/development. But it is totally fine for now